New browser install flow (#13902)
newrelic/docs-website: src/content/docs/browser/browser-monitoring/installation/install-browser-monitoring-agent.mdx

## Look for your data in New Relic [#generate-traffic]

When you're done installing the browser agent, wait a few minutes and see if data is present in [our browser monitoring UI](/docs/browser/browser-monitoring/getting-started/browser-summary-page).

If it's been more than 15 minutes and you're not seeing data, see [Troubleshooting](/docs/browser/browser-monitoring/troubleshooting/troubleshoot-your-browser-monitoring-installation/).

## What's next? [#whats-next]

Ideas for next steps: 

* View your app data on the [Summary page](/docs/browser/browser-monitoring/getting-started/browser-summary-page).
* Now that you've instrumented with the browser agent, go deeper with your data with our [Improve your website tutorial](/docs/journey-performance/guide-to-monitoring-core-web-vitals). 
* Learn how to optimize your browser monitoring experience with our [Best practices guide for browser monitoring](/docs/new-relic-solutions/best-practices-guides/full-stack-observability/browser-monitoring-best-practices-guide). 

## Install methods [#choose-deployment]

When it comes to adding the browser agent to your webpages, you have two deployment methods: 

> * APM + browser monitoring: you can use an APM agent to get browser monitoring 

Reviewer: ```suggestion
* APM + browser monitoring: Use one of our APM agents to auto-instrument browser monitoring 
```
Something like that for consistency/clarity

* Browser monitoring only: Add our JavaScript snippet to your front-end HTML 

Here's some guidance on these two methods. 

> ### Use APM agent [#apm-agent]

Reviewer: ```suggestion
### APM + browser monitoring [#apm-agent]
```

* [PHP](/docs/agents/php-agent/features/page-load-timing-php)
* [Python](/docs/agents/python-agent/supported-features/page-load-timing-python)
* [Ruby](/docs/agents/ruby-agent/features/page-load-timing-ruby)

> ### Add JavaScript code [#copy-paste]

Reviewer: ```suggestion
### Browser monitoring only [#copy-paste]
```

Reviewer: For consistency with the bullet points previously

> Use this method if you can't use APM for whatever reason (for example, if you have a [SPA](/docs/browser/single-page-app-monitoring/get-started/introduction-single-page-app-monitoring) app, or if your APM agent isn't supported for auto-injection, as is true for Node.js and some Java apps). When you go this route, you'll receive a customized JavaScript snippet at the end of the install process in the New Relic UI, and you'll add that to the HTML of any pages you want to monitor in your app/site.

Reviewer: ```suggestion
Use this method if you can't use APM for whatever reason, like if you have a [SPA](/docs/browser/single-page-app-monitoring/get-started/introduction-single-page-app-monitoring) app, or if your APM agent isn't supported for auto-injection (as is true for Node.js and some Java apps). If you use this method, you'll receive a customized JavaScript snippet at the end of the install process in the New Relic UI, and you'll add that to the HTML of any pages you want to monitor in your app/site.
```
